data/pools/transactionPool.go

```diff
@@ -665,6 +665,10 @@ func (pool *TransactionPool) recomputeBlockEvaluator(committedTxIds map[transact
 
 	next := bookkeeping.MakeBlock(prev)
 	pool.numPendingWholeBlocks = 0
+	hint := pendingCount - int(knownCommitted)
```

Comment:
I think that you meant to pass this `hint` to the `pool.ledger.StartEvaluator` but forgot to do so.

Reply:
thank you fixed